### PR TITLE
Fix example script path

### LINKED ISSUE #271: Fix example script
**Describe the bug**
The `arc_01` example doesn't work due to a broken path.

**To Reproduce**
Run the example code with the instructions on the website.

**Expected behaviour**
It should run and produce the plots.

### PR DESCRIPTION
**Pull request by willGraham01**
# Context/Description

* Fixes #271 

## More details

Updates the additional path added to the `MATLABPATH` when running the example script, so that MATLAB can find the required files.

## Testing

`examples/arc_01` can now be run using the instructions in the README.md and does not error.

### CODE REVIEW

File: examples/arc_01/run_pstd_bscan.m [hunk 1]
```diff
@@ -56,3 +56,8 @@
 imagesc(dat_fs.z_i,dat_fs.x_i,abs(squeeze(dat_cyl.Ex_i)));
 title('Focussed beam with scattering cylinder');
 axis equal;
+
+%%
+figure;
+imagesc(dat_cyl.z_i, dat_cyl.x_i, abs(squeeze(dat_cyl.Hz_out)));
+axis equal;
```
Comment (samcunliffe) [inline examples/arc_01/run_pstd_bscan.m:63]: This makes a second image, does it?
+1 to leaving it in though, if it's the nice sphere one.

**Review by samcunliffe — APPROVED** [review]
🍉

File: examples/arc_01/run_pstd_bscan.m (hunk 1 above)
Comment (willGraham01) [reply examples/arc_01/run_pstd_bscan.m:63]: It makes our sphere logo in fact!

Comment (samcunliffe) [reply examples/arc_01/run_pstd_bscan.m:63]: What is flavour text?
I mean sure: go nuts.

**Pull request merged**